src/rp2_common/pico_stdio_usb/include/pico/stdio_usb.h

```diff
@@ -54,6 +54,18 @@
 #define PICO_STDIO_USB_RESET_MAGIC_BAUD_RATE 1200
 #endif
 
+// Setting this to something other than -1 will prevent picotool specifying an activity LED via VENDOR_INTERFACE. Set to -2 to specify none whilst also preventing picotool from specifying an activity LED.
```

Comment:
so again get rid of the -1 and use undefined, and split the -2 off into a separate define to disallow overriding the value via the reset interface

Reply:
I think I've done what you were asking for :slightly_smiling_face: